src/main/kotlin/eu/europa/ec/eudi/openid4vci/Issuance.kt

```diff
@@ -235,40 +156,6 @@ interface RequestBatchIssuance {
     suspend fun AuthorizedRequest.requestBatchAndUpdateState(
```

Comment:
Shall this be renamed to AuthorizedRequest.requestBatch()? Since the default behavior is to update the state internally I guess there is no need to state it on the method name